packages/template/src/project.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
This file shouldn't be committed

Reply:
Should i just delete this file in my new commit or should i revert back to the previous HEAD and not commit this files.

Reply:
Whatever works for you. We use "squash and merge" in this repo to keep linear history so all your changes will be turned into a single commit regardless.